scripts/set-react-version.js

```diff
@@ -33,28 +33,19 @@ const profiles = {
     "react-native-windows": "0.63.14",
   },
   0.64: {
-    "@react-native-community/cli": "^5.0.1-alpha.0",
-    "@react-native-community/cli-platform-android": "^5.0.1-alpha.0",
-    "@react-native-community/cli-platform-ios": "^5.0.1-alpha.0",
```

Comment:
I would argue that we should have >= 5 here...

Reply:
`react-native` pulls in the correct version if we just omit it. That's why I removed it here, and below.